x-pack/test/api_integration/apis/security/security_basic.ts

```diff
@@ -9,6 +9,7 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 
 export default function ({ loadTestFile }: FtrProviderContext) {
   describe('security (basic license)', function () {
+    this.tags('skipFIPS');
```

Comment:
Just to confirm...are our API tests in being performed in FIPS mode as part of the common config and general index file?

Reply:
Correct! Fortunately the files were mirrored so we get full coverage w/o basic